src/rgw/driver/d4n/d4n_policy.cc

```diff
@@ -438,6 +443,7 @@ void LFUDAPolicy::update(const DoutPrefixProvider* dpp, const std::string& key,
   bool updateLocalWeight = true;
   uint64_t refcount = 0;
 
+  ldpp_dout(dpp, 10) << "\n" <<  __LINE__ << ": " << key << "\n" << dendl;
```

Comment:
are the keys printed in this method correct, when you do not run them under gdb?